ttnn/cpp/ttnn/operations/eltwise/binary/device/binary_composite_op.cpp

```diff
@@ -392,4 +392,13 @@ Tensor _polyval(const Tensor& input_a, const std::vector<float>& coeffs, const s
     return final_tensor;
 }
 
+Tensor ExecuteBinaryLE::operator()(const Tensor& input_a, const Tensor& input_b, const std::optional<MemoryConfig>& output_mem_config) {
+    return ttnn::le(input_a, input_b, std::nullopt, output_mem_config, input_a);
+}
+
+
+Tensor ExecuteBinaryLE::operator()(const Tensor& input, float scalar, const std::optional<MemoryConfig>& output_mem_config) {
+    return ttnn::le(input, scalar, std::nullopt, output_mem_config, input);
+}
+
```

Comment:
Why does this operation still take in output mem config when it is specifically an in place operation so output_mem_config will never actually be needed/used since it will always be the input mem config?